test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -263,24 +263,24 @@
     (testing "returns a 400 when an invalid level is queried for"
       (let [resp (http-client/get "http://localhost:8180/status/v1/services?level=bar")]
         (is (= 400 (:status resp)))
-        (is (= {"type" "request-data-invalid"
-                "message" "Invalid level: :bar"}
+        (is (= {"error" {"type" "request-data-invalid"
+                         "message" "Invalid level: :bar"}}
```

Comment:
I feel like the next release of tk-status should indicate a breaking change and this should be fine?